ui/mirage/factories/project.ts

```diff
@@ -14,6 +14,18 @@ export default Factory.extend({
     remoteEnabled: true,
   }),
 
+  'with-input-variables': trait({
+    name: 'with-input-variables',
+
+    afterCreate(project, server) {
+      let variables = [
+        server.create('variable', 'random-str', { project }),
+        server.create('variable', 'random-str', { project }),
+        server.create('variable', 'random-hcl', { project }),
+      ];
```

Comment:
Since we’re not doing anything further with this variables in this function, we can drop the referencing array.

```suggestion
      server.createList('variable', 2, 'random-str', { project });
      server.create('variable', 'random-hcl', { project });
```